tests/snapd

```diff
@@ -5,9 +5,6 @@ set -eux
 # shellcheck disable=SC1091
 [ -e "/etc/os-release" ] && . /etc/os-release
```

Comment:
This can go as it was used for `[ "${ID}" != "ubuntu-core" ]` which is no longer there.